### DISCUSSION TITLE
How to disable render on all files in a directory

### DISCUSSION THREAD

**Discussion by grepinsight** [quarto-dev/quarto-cli#5672] [Q&A] [answered]
I have a bunch of Rmd files from years ago that I want to keep in a directory in the repo I use for Quarto. Is there a configuration option that will disable rendering for all documents in a folder? I attempted to use `_metadata.yml`, to no avail.

```yml
execute:
  enabled: false
```

**Comment by mcanouil** [comment, marked as answer]
See <https://quarto.org/docs/websites/index.html#render-targets>.
